DEVELOPER.md

```diff
@@ -14,10 +14,13 @@ See http://kafka.apache.org/documentation.html#consumerconfigs for details about
 
     input {
         kafka {
-            topic_id => ... # string (required), The topic to consume messages from
+            topic_id => ... # string (optional), default: nil, The topic to consume messages from. Can be a java regular expression for whitelist of topics.
```

Comment:
Ok @talevy pointed me to this line https://github.com/logstash-plugins/logstash-input-kafka/pull/25/files#diff-f991873fca4b096b3ff166fd7027a915R107 which checks for config